### ISSUE TITLE
Li ccECP_reg UPF file is incorrect

### ISSUE THREAD

**Issue by bruneval** [QMCPACK/pseudopotentiallibrary#77]
Dear pseudizers,

Li ccECP_reg UPF pseudopotential file contains
```
  <PP_DIJ>
 1   Number of nonzero Dij
 1 1                 inf
  </PP_DIJ>
```
Obviously `inf` is not nice for ABINIT.
Could you please fix it?

Fabien

**Comment by prckent** [comment]
Yes we will. Sorry! Clearly some edge case in a script it not right...

**Comment by bruneval** [comment]
Thank you Paul!

Keep me posted. I think these pseudos would be very useful for my GW calculations.

Fabien

**Comment by prckent** [comment]
He and Be also appear to have this issue

**Comment by bruneval** [comment]
Thank you guys. Nice work. I'm eager to try your pseudos.

Fabien
